geopandas/geodataframe.py

```diff
@@ -214,10 +214,33 @@ def __setattr__(self, attr, val):
 
     def _get_geometry(self):
         if self._geometry_column_name not in self:
-            raise AttributeError(
-                "No geometry data set yet (expected in"
-                " column '%s'.)" % self._geometry_column_name
-            )
+            if self._geometry_column_name is None:
+                msg = (
+                    "You are calling a geospatial method on the GeoDataFrame, "
+                    "but the default active geometry column to use has not been set. "
```

Comment:
```suggestion
                    "but the active geometry column to use has not been set. "
```
I'd say that "default" means pre-set, which does not make sense in this context.